### PR TITLE
Merge Bundles into a single namespace

### LINKED ISSUE #322: Merge bundles together into a single AppBundle
From Symfony documentation

> ...it was recommended to organize your own application code using bundles. This is no longer recommended and bundles should only be used to share code and features between multiple applications.

We have very little code to be put in their own bundles, and bundle splitting just forces us to look up different files for configuration. Merging them into a single bundle will bring all configuration to a single place.

Effect on code quality will not be much IMO, the existing controller and services continue to exist as they were and no additional coupling would be introduced.

### PR DESCRIPTION
**Pull request by agathver**
One step closer to Symfony 4.0 & Flex

All code now lives under `App\*`

Closes #322 

## Implementation

- [x] Entities
- [x] Repositories
- [x] Controllers
- [x] Service classes
- [x] Commands
- [x] RepoCrawler
- [x] Utils
- [x] Fixtures
- [x] Third-party configurations
- [x] Service config
- [x] Routing config
- [x] Templates
- [x] Forms
- [x] Tests

## Quality Assurance Checklist

- [x] Basic selenium tests run

### Basic 
- [x] Home page renders
- [x] Browse projects when logged out
- [x] Login through email
- [x] Login through Github
- [x] Add project
- [x] Crawler crawls the project successfully
- [x] Project metrics are displayed
- [x] Static pages are rendered

### User management and login
- [x] Register
- [x] Password Reset
- [x] Login through Google
- [x] Change password
- [x] Change name

### Projects
- [x] Browse projects
- [x] Add GitHub project
- [x] Algolia index is updated succesfully
- [x] Change project settings
- [x] Add organization
- [x] Add project to organization

### CODE REVIEW

File: site/.editorconfig [hunk 1]
```diff
@@ -2,15 +2,11 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 2
+indent_size = 4
```
Comment (imphil) [inline site/.editorconfig]: We indent HTML, CSS and JS files with two spaces (iirc).

Comment (agathver) [reply site/.editorconfig]: We use 2 for html, css, js and 4 elsewhere where not specified

**Comment by imphil** [comment]
@agathver can you do one last rebase on this one and then I'll merge it as next thing?

**Comment by agathver** [comment]
You will need to run `librecores:update-repos` after this checking out this code. 

5d90c88 resets all array columns to empty values to a avoid deserialization errors.

**Comment by imphil** [comment]
Thanks! This is now merged and we can build on top of it!

**Pull request merged**